### DISCUSSION TITLE
The manual is very pool and inaccurate

### DISCUSSION THREAD

**Discussion by mrken277** [lowcoder-org/lowcoder#354] [General]
Dear Team 
i feeling very bad with your docs, i try many times to build the sdk but still can not even try with cloud and self-host backend-server. while do follow your suggestions i also saw many bug , some need to install correct nodejs version, some about webback config , or using diff api url between package node and test html page... and until now after using OpenBlocks then Low-coder for 6 months i still can not build SDK. I need to use SDK to intergrant with current system platform >:< Beside it i also got some issue with performarce while try heavy api, our system using very high server spec so i think it cause by web browser but i can not get how to use server rendering 
Note: i have exp with many other low-code platform and this comment get after review many other platorm docs, right now i also still use both lowcoder and tooljet

**Comment by FalkWolsky** [comment]
Are they better in the meantime? 

We would feel honored if you could contribute to the docs too.
